src/EnergyPlus/EarthTube.cc

```diff
@@ -583,55 +569,50 @@ void CalcEarthTube(EnergyPlusData &state)
+             std::abs(state.dataEnvrn->OutDryBulbTemp - state.dataHeatBalFanSys->MAT(NZ)) * thisEarthTube.TemperatureTermCoef +
+             state.dataEnvrn->WindSpeed * (thisEarthTube.VelocityTermCoef + state.dataEnvrn->WindSpeed * thisEarthTube.VelocitySQTermCoef));
 
         state.dataHeatBalFanSys->EAMFL(NZ) = state.dataHeatBalFanSys->MCPE(NZ) / AirSpecHeat;
-        if (state.dataEarthTube->EarthTubeSys(Loop).FanEfficiency > 0.0) {
-            state.dataEarthTube->EarthTubeSys(Loop).FanPower = state.dataHeatBalFanSys->EAMFL(NZ) *
-                                                               state.dataEarthTube->EarthTubeSys(Loop).FanPressure /
-                                                               (state.dataEarthTube->EarthTubeSys(Loop).FanEfficiency * AirDensity);
+        if (thisEarthTube.FanEfficiency > 0.0) {
+            thisEarthTube.FanPower = state.dataHeatBalFanSys->EAMFL(NZ) * thisEarthTube.FanPressure / (thisEarthTube.FanEfficiency * AirDensity);
         }
 
-        AverPipeAirVel = EVF / DataGlobalConstants::Pi / pow_2(state.dataEarthTube->EarthTubeSys(Loop).r1);
+        AverPipeAirVel = EVF / DataGlobalConstants::Pi / pow_2(thisEarthTube.r1);
         AirMassFlowRate = EVF * AirDensity;
 
         // Calculation of Average Ground Temperature between Depth z1 and z2 at time t
-        GroundTempz1z2t = state.dataEarthTube->EarthTubeSys(Loop).AverSoilSurTemp -
-                          state.dataEarthTube->EarthTubeSys(Loop).ApmlSoilSurTemp *
-                              std::exp(-state.dataEarthTube->EarthTubeSys(Loop).z *
-                                       std::sqrt(DataGlobalConstants::Pi / 365.0 / state.dataEarthTube->EarthTubeSys(Loop).SoilThermDiff)) *
+        GroundTempz1z2t = thisEarthTube.AverSoilSurTemp -
+                          thisEarthTube.ApmlSoilSurTemp *
+                              std::exp(-thisEarthTube.z * std::sqrt(DataGlobalConstants::Pi / 365.0 / thisEarthTube.SoilThermDiff)) *
                               std::cos(2.0 * DataGlobalConstants::Pi / 365.0 *
-                                       (state.dataEnvrn->DayOfYear - state.dataEarthTube->EarthTubeSys(Loop).SoilSurPhaseConst -
-                                        state.dataEarthTube->EarthTubeSys(Loop).z / 2.0 *
-                                            std::sqrt(365.0 / DataGlobalConstants::Pi / state.dataEarthTube->EarthTubeSys(Loop).SoilThermDiff)));
-        state.dataEarthTube->EarthTubeSys(Loop).GroundTempz1z2t = GroundTempz1z2t;
+                                       (state.dataEnvrn->DayOfYear - thisEarthTube.SoilSurPhaseConst -
+                                        thisEarthTube.z / 2.0 * std::sqrt(365.0 / DataGlobalConstants::Pi / thisEarthTube.SoilThermDiff)));
+        thisEarthTube.GroundTempz1z2t = GroundTempz1z2t;
 
         // Calculation of Convective Heat Transfer Coefficient at Inner Pipe Surface
         AirThermCond = 0.02442 + 0.6992 * state.dataEnvrn->OutDryBulbTemp / 10000.0;
```

Comment:
These are some fun constants. Should they be symbolic `constexpr` somewhere? Or just leave them like this?